keel-ec2-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ec2.kt

```diff
@@ -16,11 +16,13 @@
 package com.netflix.spinnaker.keel.api.ec2
 
 import com.netflix.spinnaker.keel.api.ec2.old.ApplicationLoadBalancerV1Spec
+import com.netflix.spinnaker.keel.api.ec2.old.ClusterV1Spec
 import com.netflix.spinnaker.keel.api.plugins.kind
 
 const val CLOUD_PROVIDER = "aws"
 
-val EC2_CLUSTER_V1 = kind<ClusterSpec>("ec2/cluster@v1")
+val EC2_CLUSTER_V1_1 = kind<ClusterSpec>("ec2/cluster@v1.1")
```

Comment:
Side-note: we really need a way to generate model docs from code based on the schema that documents all supported versions and indicates which one is the canonical version. Every time we add a new version, our manually-crafted docs become outdated.

Reply:
I was thinking we should maybe even consider removing the older ones from the schema, or including them on a different endpoint or something.